### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by ShowMeYourSkil**
Good morning,

I have upgraded your readme. Now it is more informative, nicer and just better. 
Since I'm not you and don't know exactly what you want it to say, I wrote in what I think you want to read. If something is doff or missing you can update it after the pull request.

### CODE REVIEW

**Comment by mtorials** [comment]
Ouh here is still the virtualbox aricle in!

**Comment by ShowMeYourSkil** [comment]
> Ouh here is still the virtualbox aricle in!

Ups. My new branch struktur for all pull_request is new :D i have already remove the article for you

File: README.md [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
-
-### Content
-
-The content of this website is licesed under the [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.
-Content means the actual posts and texts under the `content` directory.
+<img src="https://user-images.githubusercontent.com/39274150/111981613-30d1ff00-8b08-11eb-8767-b89f2e0970be.png"/>
+<h1>https://mt32.net</h1>
+<h2>The blog for all computer interested</h2>
+
+ Netlify
```
Comment (mtorials) [inline README.md]: mt32.net is no longer deployed with netlify

File: README.md [hunk 2]
```diff
@@ -1,16 +1,42 @@
...
+<h1>https://mt32.net</h1>
+<h2>The blog for all computer interested</h2>
+
+ Netlify  
+|----|
+[![Netlify Status](https://api.netlify.com/api/v1/badges/4e30a997-7aa8-4fa2-8856-93e9b9a5ee39/deploy-status)](https://app.netlify.com/sites/mtorials-test/deploys)
+
+# About mt32.net
+
+`mt32.net` is running by @mtorials and @universum two hobby programmer.
```
Comment (mtorials) [inline README.md]: I am the only one working on this....

File: README.md [hunk 3]
```diff
@@ -1,16 +1,42 @@
...
+# Write your own Article 
+## Write your own articles with markdown
+![image](https://user-images.githubusercontent.com/39274150/111983370-59f38f00-8b0a-11eb-9b6b-377c71e16d35.png)
+
+# How to write a article?
+
+Do you want make a article! Nice! Please read THIS file when you're new and do not know how.
+
+# Netifly
+
```
Comment (mtorials) [inline README.md]: again this is no longer the case

File: README.md (hunk 2 above)
Comment (ShowMeYourSkil) [reply README.md]: i was use the source on your about page. You should update this site.

File: README.md (hunk 1 above)
Comment (ShowMeYourSkil) [reply README.md]: Okey thank you. I want to remove netlify!

File: README.md (hunk 2 above)
Comment (ShowMeYourSkil) [reply README.md]: I was writen from your about source. https://mt32.net/about. I will remove universum.

File: README.md (hunk 3 above)
Comment (ShowMeYourSkil) [reply README.md]: Okey. I will remove them.

**Review by ShowMeYourSkil — COMMENTED** [review]
@mtorials i have meet your wishes. Now it is okey for you?

**Pull request merged**